source/UIAHandler/browseMode.py

```diff
@@ -160,6 +174,7 @@ def UIAHeadingQuicknavIterator(
 		direction: str = "next"
 ):
 	reverse = bool(direction == "previous")
+	wantedLevel = int(itemType[7:]) if len(itemType) > 7 else None

```

Comment:
It would be a bit cleaner to use a constant instead of the index 7 (e.g. the length of the word 'heading').